src/coreclr/jit/emitarm64.cpp

```diff
@@ -14662,6 +14804,219 @@ void emitter::emitIns_Call(EmitCallType          callType,
+
+        case IF_SVE_HW_4B:
+            assert(insSveIsLsl(ins, fmt));
+            switch (ins)
+            {
+                case INS_sve_ld1h:
+                case INS_sve_ld1sh:
+                case INS_sve_ldff1h:
+                case INS_sve_ldff1sh:
+                    return 1;
+
+                case INS_sve_ld1w:
+                case INS_sve_ldff1w:
+                    return 2;
+
+                default:
+                    break;
+            }
+            assert(!"Invalid instruction");
+            return 0;
+
+        case IF_SVE_JJ_4A:
+        case IF_SVE_JJ_4A_B:
+        case IF_SVE_JJ_4A_C:
+        case IF_SVE_JJ_4A_D:
+        case IF_SVE_JK_4A:
+        case IF_SVE_JK_4A_B:
+        case IF_SVE_HW_4A:
+        case IF_SVE_HW_4A_A:
+        case IF_SVE_HW_4A_B:
+        case IF_SVE_HW_4A_C:
+        case IF_SVE_IU_4A:
+        case IF_SVE_IU_4A_A:
+        case IF_SVE_IU_4A_C:
+            switch (ins)
+            {
+                case INS_sve_ld1h:
+                case INS_sve_ld1sh:
+                case INS_sve_ldff1h:
+                case INS_sve_ldff1sh:
+                    switch (fmt)
+                    {
+                        case IF_SVE_HW_4A:
+                        case IF_SVE_HW_4A_A:
+                            return 1;
+
+                        default:
+                            break;
+                    }
+                    return 0;
+
+                case INS_sve_ld1w:
+                case INS_sve_ldff1w:
+                case INS_sve_ld1sw:
+                case INS_sve_ldff1sw:
+                    switch (fmt)
+                    {
+                        case IF_SVE_HW_4A:
+                        case IF_SVE_HW_4A_A:
+                        case IF_SVE_IU_4A:
+                            return 2;
+
+                        default:
+                            break;
+                    }
+                    return 0;
+
+                case INS_sve_ld1d:
+                case INS_sve_ldff1d:
+                    switch (fmt)
+                    {
+                        case IF_SVE_IU_4A:
+                            return 3;
+
+                        default:
+                            break;
+                    }
+                    return 0;
+
+                case INS_sve_st1h:
+                    switch (fmt)
+                    {
+                        case IF_SVE_JJ_4A_C:
+                        case IF_SVE_JJ_4A_D:
+                            return 0;
+
+                        default:
+                            break;
+                    }
+                    return 1;
+
+                case INS_sve_st1w:
+                    switch (fmt)
+                    {
+                        case IF_SVE_JJ_4A_C:
+                        case IF_SVE_JJ_4A_D:
+                            return 0;
+
+                        default:
+                            break;
+                    }
+                    return 2;
```

Comment:
i would suggest to explicitly call out the formats in case and for everything else just `assert`, same as what I suggested above.